config/example.database.yml

```diff
@@ -8,6 +8,7 @@ development:
 #  password: openstreetmap
 #  host: localhost
   encoding: utf8
+  pool: <%= ENV["DB_POOL"] || ENV['RAILS_MAX_THREADS'] || 5 %>
```

Comment:
Your PR does not explain why you are making this change, which affects all users not just docker users.